_includes/gc-follow-us.html

```diff
@@ -1,9 +1,17 @@
 <section class="provisional gc-followus">
 	<h2>
-		{%- if i18nText-lang == "fr" -%}
-			Dans les médias sociaux
+		{%- if include.inlineLayout -%}
```

Comment:
Please call it `isInline`, since we consider it good practice to prefix boolean variables with `is`.
```suggestion
		{%- if include.isInline -%}
```